sandbox/tests/test scenes/standard surface next/01 - hazy gloss.appleseed

```diff
@@ -0,0 +1,278 @@
+                <parameter name="importance_multiplier" value="1.0" />
+                <parameter name="light_near_start" value="0.0" />
+                <parameter name="radiance" value="orange_light_color" />
+                <parameter name="radiance_multiplier" value="3" />
+            </edf>
+            <edf name="right_light_edf" model="diffuse_edf">
+                <parameter name="cast_indirect_light" value="true" />
+                <parameter name="importance_multiplier" value="1.0" />
+                <parameter name="light_near_start" value="0.0" />
+                <parameter name="radiance" value="blue_light_color" />
+                <parameter name="radiance_multiplier" value="3" />
+            </edf>
+            <edf name="top_light_edf" model="diffuse_edf">
+                <parameter name="cast_indirect_light" value="true" />
+                <parameter name="importance_multiplier" value="1.0" />
+                <parameter name="light_near_start" value="0.0" />
+                <parameter name="radiance" value="1" />
+                <parameter name="radiance_multiplier" value="2" />
+            </edf>
+            <surface_shader name="surface_shader" model="physical_surface_shader">
+                <parameter name="lighting_samples" value="1" />
+            </surface_shader>
+            <material name="diffuse_material" model="generic_material">
+                <parameter name="bsdf" value="diffuse_brdf" />
+                <parameter name="bump_amplitude" value="1.0" />
+                <parameter name="displacement_method" value="bump" />
+                <parameter name="normal_map_up" value="z" />
+                <parameter name="surface_shader" value="surface_shader" />
+            </material>
+            <material name="left_light_material" model="generic_material">
+                <parameter name="bsdf" value="diffuse_brdf" />
+                <parameter name="bump_amplitude" value="1.0" />
+                <parameter name="displacement_method" value="bump" />
+                <parameter name="edf" value="left_light_edf" />
+                <parameter name="normal_map_up" value="z" />
+                <parameter name="surface_shader" value="surface_shader" />
+            </material>
+            <material name="right_light_material" model="generic_material">
+                <parameter name="bsdf" value="diffuse_brdf" />
+                <parameter name="bump_amplitude" value="1.0" />
+                <parameter name="displacement_method" value="bump" />
+                <parameter name="edf" value="right_light_edf" />
+                <parameter name="normal_map_up" value="z" />
+                <parameter name="surface_shader" value="surface_shader" />
+            </material>
+            <shader_group name="shaderball_mat_shader_group">
+                            <shader type="shader" name="as_standard_surface_next" layer="shaderball_mat">
```

Comment:
These files should be loaded and re-saved by appleseed to fix the formatting. You can also use the `projecttool` command line utility that ships with appleseed.